dali/test/python/test_utils_tensorflow.py

```diff
@@ -88,6 +88,36 @@ def expect_iter_end(should_raise, exception_type):
 #
 # ################################################################################################ #
 
+def get_mix_size_image_pipeline(batch_size, num_threads, device, device_id=0, shard_id=0, num_shards=1,
+        def_for_dataset=False):
+    test_data_root = os.environ['DALI_EXTRA_PATH']
```

Comment:
Why not use `get_dali_extra_path()` from test_utils.py?

Reply:
I reused part of a function that used this. Updated in both.